expat/lib/xmlparse.c

```diff
@@ -6240,7 +6240,7 @@ storeEntityValue(XML_Parser parser, const ENCODING *enc,
           dtd->keepProcessing = dtd->standalone;
           goto endEntityValue;
         }
-        if (entity->open) {
+        if (entity->open || entity == parser->m_declEntity) {
```

Comment:
If this is consistent with your coding style OK, but I always like to group conditions like this with parentheses, `(entity->open || (entity == parser->m_declEntity))`

Reply:
@timbray alright, adjusted and pushed. Thanks for the review!